### ISSUE TITLE
[A11y] Dashboard - Headings

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1847]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

The navigation sidebar doesn't have any headings.  The username ("admin@example.org") somewhat functions as a heading, but isn't coded as one.  The sections within the sidebar function as headings as well (e.g., "ACTIVITY", "REPOSITORY CONTENTS", etc.), but also are not coded as headings.

### Suggested Fix

The username piece of the sidebar could be a level 2 heading.  The user's email address is kind of an odd heading, but that's a usability issue at worst - it's the same for sighted people.  But as it functions as a heading for sighted users, it should do the same for assistive tech users.

The sidebar sections should be level-3 headings.

**Comment by jechols** [comment]
QA: Inspect elements / view source, or use a tool to view an outline by headings

**Comment by CGillen** [comment]
Fixed by #2009
